### PR TITLE
ppe42-gcc latest update

### PR DESCRIPTION
**Pull request by dgilbert999**

### CODE REVIEW

**Comment by stewart-ibm** [comment]
Can one of the admins verify this patch for testing?

**Comment by stewart-ibm** [comment]
ok to test

**Comment by stewart-ibm** [comment]
retest this please

File: openpower/package/ppe42-gcc/0001-2016-02-19-Jakub-Jelinek-jakub-redhat.com.patch [hunk 1]
```diff
@@ -52,7 +52,7 @@ index 2784edc..8d7f015 100644
 -#endif
 -static unsigned int hash (const char *, unsigned int);
 -#ifdef __GNUC__
--__inline
+-//__inline
```
Comment (stewart-ibm) [inline openpower/package/ppe42-gcc/0001-2016-02-19-Jakub-Jelinek-jakub-redhat.com.patch]: why this change?

Isn't this a backported fix? Why would it suddenly change?

File: openpower/package/ppe42-gcc/ppe42-gcc.mk [hunk 2]
```diff
@@ -4,7 +4,7 @@
 #
 ################################################################################
 
-PPE42_GCC_VERSION ?= d8a1bac8634033a3edd4e9a22455f97318718f43
+PPE42_GCC_VERSION ?= f8c463f16d47773cfd9946a8948a50efde3fee14
```
Comment (stewart-ibm) [inline openpower/package/ppe42-gcc/ppe42-gcc.mk]: can you please include a "git shortlog d8a1bac8634033a3edd4e9a22455f97318718f43..f8c463f16d47773cfd9946a8948a50efde3fee14" in the commit message so we can easily spot from git history of op-build what has changed?

File: openpower/package/ppe42-gcc/0001-2016-02-19-Jakub-Jelinek-jakub-redhat.com.patch (hunk 1 above)
Comment (dgilbert999) [reply openpower/package/ppe42-gcc/0001-2016-02-19-Jakub-Jelinek-jakub-redhat.com.patch]: Because I also made changes in this area in a previous version

**Comment by dgilbert999** [comment]
There is another change to the compiler.
